src/Movie/Movie.js

```diff
@@ -13,20 +13,20 @@ const Movie = ({ movieDetails }) => {
             <div className='movie-heading-container' 
                 style={{backgroundImage: `url(${backdropPath})`, 
                 backgroundSize: 'cover',
-                boxShadow: 'inset 30vw 10px 100px black'
+                boxShadow: 'inset 40vw 10px 100px black'
             }}>
                 <h2>{title}</h2>
                 <ul className="movieHeadingDetails">
-                    <li>Average Rating: {averageRating}</li>
-                    <li>{genres.join(' | ')} {releaseDateFormatted}</li>
+                    <li>Average Rating: {averageRating.toFixed(2)}</li>
+                    <li>{genres.join(' | ')} Released: {releaseDateFormatted}</li>
                     <li>{runtime} Minutes</li>
                     <li>Budget: ${budget.toLocaleString('en-US')}</li>
                     <li>Revenue: ${revenue.toLocaleString('en-US')}</li>
                 </ul>
             </div>
             <div className="movie-description-container">
                 <div>
-                    {tagline.length && <h3 className="tagline">{tagline}</h3>}
```

Comment:
This was printing '0' when there was no tagline lol so using the falsy value of an empty string works better

Reply:
nice work looks great!